gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -378,6 +382,9 @@ func (vm *VMKeeper) AddPackage(ctx sdk.Context, msg MsgAddPackage) (err error) {
 	if gm.HasReplaces() {
 		return ErrInvalidPackage("development packages are not allowed")
 	}
+	if gm.Draft && ctx.BlockHeight() > 0 {
+		return ErrInvalidPackage("draft packages can only be deployed at genesis time")
+	}
```

Comment:
nit: The gnomod sanity checks should ideally run before gno.TypeCheckMemPackage to fail early.